src/coreml.cs

```diff
@@ -584,6 +754,233 @@ interface MLModelConfiguration : NSCopying, NSSecureCoding {
 
 		[Export ("computeUnits", ArgumentSemantic.Assign)]
 		MLComputeUnits ComputeUnits { get; set; }
+
+		// From MLModelConfiguration (MLGPUConfigurationOptions)
+
+		[NoWatch, TV (13,0), Mac (10,15), iOS (13,0)]
+		[Export ("allowLowPrecisionAccumulationOnGPU")]
+		bool AllowLowPrecisionAccumulationOnGpu { get; set; }
+
+		[NoWatch, TV (13,0), Mac (10,15), iOS (13,0)]
+		[NullAllowed, Export ("preferredMetalDevice", ArgumentSemantic.Assign)]
+		IMTLDevice PreferredMetalDevice { get; set; }
+
+		// From MLModelConfiguration (MLModelParameterAdditions)
+
+		[Watch (6,0), TV (13,0), Mac (10,15), iOS (13,0)]
+		[NullAllowed, Export ("parameters", ArgumentSemantic.Assign)]
+		NSDictionary<MLParameterKey, NSObject> Parameters { get; set; }
+	}
+
+	[Watch (6,0), TV (13,0), Mac (10,15), iOS (13,0)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface MLKey : NSCopying, NSSecureCoding {
+
+		[Export ("name")]
+		string Name { get; }
+
+		[NullAllowed, Export ("scope")]
+		string Scope { get; }
+	}
+
+	[Watch (6,0), TV (13,0), Mac (10,15), iOS (13,0)]
+	[BaseType (typeof (MLKey))]
+	[DisableDefaultCtor]
+	interface MLMetricKey {
+
+		[Static]
+		[Export ("lossValue")]
+		MLMetricKey LossValue { get; }
+
+		[Static]
+		[Export ("epochIndex")]
+		MLMetricKey EpochIndex { get; }
+
+		[Static]
+		[Export ("miniBatchIndex")]
+		MLMetricKey MiniBatchIndex { get; }
+	}
+
+	[Watch (6,0), TV (13,0), Mac (10,15), iOS (13,0)]
+	[BaseType (typeof (NSObject))]
```

Comment:
sounds like a candidate for `[DisableDefaultCtor]` since there are no setters, only getters - what's the point of creating an instance ?